dev-libs/blake3/files/blake3-1.5.3-backport-pr405.patch

```diff
@@ -0,0 +1,57 @@
+https://bugs.gentoo.org/935370
+https://github.com/BLAKE3-team/BLAKE3/pull/405
+
+From aad68f7f009b7e0607b3eee3b8b8e3021fa38bd7 Mon Sep 17 00:00:00 2001
+From: matoro <matoro@users.noreply.github.com>
+Date: Tue, 2 Jul 2024 19:36:45 -0400
+Subject: [PATCH] build(CMake): support running tests with ctest
+
+---
+ CMakeLists.txt | 11 +++++++++++
+ test.py        |  4 ++--
+ 2 files changed, 13 insertions(+), 2 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3a3b232d..e778e3d7 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -168,6 +168,17 @@ else()
+   BLAKE3_DISABLE_SIMD()
+ endif()
+ 
++# cmake test support
++if (BLAKE3_BUILD_TESTING)
++  find_package(Python3 REQUIRED)
```

Comment:
Sounds like you need `python-any-r1` for that. Also, I suppose that instead of patching you could probably run `test.py` directly.

Reply:
No, this needs a non-default binary to be built unfortunately which is not hooked up to cmake at all.  The python script does not build it; upstream uses a vanilla Makefile which does not respect flags.

Can I get away with maybe just `BDEPEND` on `dev-lang/python`?  It's a very small script that only uses the stdlib.